javascript/URDFLoader.js

```diff
@@ -403,23 +414,44 @@ class URDFLoader {
     }
 
     // Process the <link> nodes
-    _processLink(link, packages, path, loadMeshCb) {
+    _processLink(link, materialMap, packages, path, loadMeshCb) {
 
         const visualNodes = this.filter(link.children, n => n.nodeName.toLowerCase() === 'visual');
         const obj = new THREE.Object3D();
         obj.name = link.getAttribute('name');
         obj.isURDFLink = true;
         obj.type = 'URDFLink';
 
-        this.forEach(visualNodes, vn => this._processVisualNode(vn, obj, packages, path, loadMeshCb));
+        this.forEach(visualNodes, vn => this._processVisualNode(vn, obj, materialMap, packages, path, loadMeshCb));
 
         return obj;
 
     }
 
-    // Process the visual nodes into meshes
-    _processVisualNode(vn, linkObj, packages, path, loadMeshCb) {
+    _processMaterial(type, value, packages, path) {
+        if (type === 'color') {
+            const rgba = value.split(/\s/g)
+                .map(v => parseFloat(v));
+            return {
+                color: {
```

Comment:
This should be an instance of `THREE.Color` if we're planning to do a copy or assignment of the object to a material. Another option would be to keep it like this but use the [`THREE.Color.setRGB`](https://threejs.org/docs/#api/en/math/Color.setRGB) function when updating the material.

Thanks for bearing with me on the changes!